lib/block-patterns.php

```diff
@@ -62,3 +62,23 @@
 						<!-- /wp:columns -->',
 	)
 );
+
+for ( $i = 0; $i < 1000; $i++ ) {
```

Comment:
For demonstration purposes only. I'll remove this once the PR is approved.

Reply:
Can we remove that code, in order to get accurate numbers in the job and judge whether the PR allows us to get the previous loading numbers?